lib/commands/session.js

```diff
@@ -13,6 +13,7 @@ commands.createSession = async function (caps) {
   this.validateDesiredCaps(caps);
   this.sessionId = UUID.create().hex;
   this.caps = caps;
+  this.rawCaps = _.cloneDeep(caps); // used for reset
```

Comment:
ah yes. here was the vision for `this.caps` and `this.opts`:

`this.caps` was supposed to be exactly what the user sent in. `this.opts` starts out as any server arguments or options that the driver is instantiated with. Because we want caps to be able to override driver options, we clone caps and merge it onto `this.opts`. So throughout the driver, we should usually only ever use `this.opts`. `this.caps` exists merely as a reference to what the user _actually_ sent it and should never be mutated.